simtools/applications/validate_camera_fov.py

```diff
@@ -58,6 +58,7 @@
 from simtools.configuration import configurator
 from simtools.io_operations import io_handler
 from simtools.model.telescope_model import TelescopeModel
+from simtools.visualization import plot_camera
```

Comment:
I would rather simply call it **camera**. since it is inside visualization, the **plot** is implicit. Their names used to be different when they are in the same module but as they are separate now, it is not necessary to change the name (just an opinion)

Reply:
Hesitating - having modules with the same name lead to confusion, even with they are in different subfolders.

Also thing that plot_camera describes well what the module is doing, while visualization.camera could also mean something related to e.g, a web page.